packages/ozone/src/db/schema/blob_divert_event.ts

```diff
@@ -0,0 +1,17 @@
+import { Generated } from 'kysely'
+
+export const eventTableName = 'blob_divert_event'
+
+export interface BlobDivertEvent {
+  id: Generated<number>
+  subjectDid: string
+  subjectBlobCid: string
+  subjectUri: string
+  divertedAt: Date | null
```

Comment:
I think we can name this `confirmedAt` to keep it consistent with `repo_push_event`, `record_push_event`, `blob_push_event`.  The idea is that once the event has been accepted by the other service it's "confirmed."